README.md

```diff
@@ -0,0 +1,68 @@
+# OFirehose
+
+Firehose for the federated social web
+
+## Installation
+
+### Prerequisites
+
+You'll need four things to get started:
+
+* node.js 4.x or higher
+* npm 1.1.0 or higher
+
+### Local install
```

Comment:
This is great. We should file a followup issue to document npm install when we publish.

Reply:
yes